src/web/assets/cp/src/css/_cp.scss

```diff
@@ -701,12 +701,12 @@ li.breadcrumb-toggle-wrapper {
     height: 30px;
     max-width: none;
     border-radius: 50%;
-    box-shadow: 0 0 0 1px transparentize($grey900, 0.95),
+    box-shadow: 0 0 0 1px transparentize($grey900, 0.6),
       0 0 0 transparentize($grey900, 1);
     transition: box-shadow linear 150ms;
 
     #user-info:hover & {
-      box-shadow: 0 0 0 1px transparentize($grey900, 0.95),
+      box-shadow: 0 0 0 1px transparentize($grey900, 0.6),
```

Comment:
Is this change necessary? It looks pretty harsh surrounding the default photo icon, and makes user photos look blurry:

<img width="187" alt="Side-by-side screenshots of the user account menu button, with the existing styles on the left, and the PR’s styles on the right." src="https://user-images.githubusercontent.com/47792/215559335-5e63f09e-8bbb-4a4c-8a06-050dc2cb7bea.png">

(Existing shadow style on the left; PR style in the right.)

Reply:
@brandonkelly it's unnecessary; I was concerned the user SVG might look strange with the original, lighter box shadow, but it actually seems fine. I've reverted it!
<img width="78" alt="Screenshot of dark gray user outline SVG with a very light gray circle border" src="https://user-images.githubusercontent.com/17525979/218146475-1483e20c-17fb-4bb5-b993-daf9b2200769.png">